### DISCUSSION TITLE
Recoil VS Jotai

### DISCUSSION THREAD

**Discussion by arch-spatula** [react-challengers/Codefolio#197] [Polls]
# recoil

## 장점

- 리팩토링 거의 없음

## 단점

- Detail 모달 hash 지원 없음(이 부분은 서비스 핵심 중 하나)

# jotai

## 장점

- Detail Modal hash 지원
    - [참고. sync-with-router](https://jotai.org/docs/guides/nextjs)
- 더욱더 간소해질 client state 코드

## 단점

- 전환비용
    - 학습기간이 필요함(리코일과 유사해도 다름)
    - 리팩토링 규모

**Comment by Jeremy-Kr** [comment]
압도적인 투표로 Recoil 현행 유지 하겠습니다.

**Reply by Jeremy-Kr** [reply]
추후 리팩토링 리소스가 남으면 다시 이슈화 시키도록 하겠습니다.
